prac_06/name_looper.py

```diff
@@ -1,39 +1,35 @@
 from kivy.app import App
 from kivy.lang import Builder
-from kivy.uix.button import Button
+from kivy.uix.button import Label
 from kivy.properties import StringProperty
 
-__author__ = 'Alan Liang'
 
-
-class NameLooperApp(App):
+class name_looper_app(App):
+    """ Main program - Kivy app to demo dynamic widget creation """
     status_text = StringProperty()
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.names = {"Bob", "Cassy", "Tom", "Sam", "Alli", "Alan", "Melina", "Chris"}
+
+        self.words = {"Bob", "Cassy", "Tom", "Sam", "Alli", "Alan", "Melina", "Chris"}
 
     def build(self):
-        self.title = "Dynamic Strings"
+        self.title = "Dynamic String"
         self.root = Builder.load_file('name_looper.kv')
         self.create_widgets()
         return self.root
 
     def create_widgets(self):
-        for name in self.names:
-            temp_button = Button(text=name)
-            temp_button.bind(on_release=self.press_entry)
-            self.root.ids.entriesBox.add_widget(temp_button)
+        for word in self.words:
+            temp_label = Label(text=word)
+            temp_label.bind(on_release=self.press_entry)
```

Comment:
Since you've copied the example, you have leftovers you weren't asked for. We only wanted the labels; not the functions. Labels don't have the `on_release` event IIRC.